.env

```
@@ -0,0 +1,2 @@
+dddd
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The dddd key should be with a value or have an equal sign

(KeyWithoutValue)